### PR TITLE
blog: cordova-android@13.0.0 release announcement

### PR DESCRIPTION
**Pull request by erisu**
Cordova Android 13.0.0 Release Announcement

### CODE REVIEW

**Review by NiklasMerz — APPROVED** [review]
LGTM. Adding the instructions to update the relevant tools is a great idea! I had some trouble using the new version and this is really helpful for every user to know.

**Review by breautek — APPROVED** [review]

File: www/_posts/2024-05-23-cordova-android-13.0.0.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    1. Open Android Studio's **SDK Manager**:
+    2. Click on `SDK Platforms` tab
+    3. Check `Android 14.0 ("UpsideDownCake")` which has the `API Level` of `34`
+    4. Click `Apply`
+
+    <img src="{{ site.baseurl }}/static/img/blog/2024/cordova-android-13-platform-sdk-34.png" style="width: 100%;" alt="Android SDK Platform" />
+
+    To install SDK Build Tools 34.0.0:
+
+    5. Open Android Studio's **SDK Manager**:
```
Comment (dpogue) [inline www/_posts/2024-05-23-cordova-android-13.0.0.md:54]: Feels a bit weird to me that the numbering keeps going from the previous list. Maybe this should start at 1 again?

File: www/_posts/2024-05-23-cordova-android-13.0.0.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    If you or a plugin has made changes to any of the following configuration preferences, the build results might not match the expected outcomes with this release:
+
+    * `android-minSdkVersion`
+    * `android-maxSdkVersion`
+    * `android-targetSdkVersion`
+    * `android-compileSdkVersion`
+    * `android-buildToolsVersion`
+    * `GradleVersion`
+    * `AndroidGradlePluginVersion`
+    * `GradlePluginKotlinVersion`
```
Comment (dpogue) [inline www/_posts/2024-05-23-cordova-android-13.0.0.md:80]: Is it worth mentioning what the default values are, or linking to where they are all defined?

We're basically telling people "If you edited any of these, please review the new settings" but not giving any pointer to where to find those new settings.

**Review by dpogue — COMMENTED** [review]
Overall looks good, I definitely agree that having more detailed instructions about how to update the tooling is a great idea.

File: www/_posts/2024-05-23-cordova-android-13.0.0.md (hunk 2 above)
Comment (breautek) [reply www/_posts/2024-05-23-cordova-android-13.0.0.md:80]: Perhaps we can give a table:

|Preference|Default Value|
|---|---|
|`android-minSdkVersion`|24|
|`android-maxSdkVersion`|_Not set_|
|`android-targetSdkVersion`|34|
|`android-compileSdkVersion`|_android-targetSdkVersion configured value_|
|`android-buildToolsVersion`|"34.0.0"|
|`GradleVersion`|"8.7"|
|`AndroidGradlePluginVersion`|"8.3.0"|
|`GradlePluginKotlinVersion`|"1.9.24"|

Markdown:

```markdown
|Preference|Default Value|
|---|---|
|`android-minSdkVersion`|24|
|`android-maxSdkVersion`|_Not set_|
|`android-targetSdkVersion`|34|
|`android-compileSdkVersion`|_android-targetSdkVersion configured value_|
|`android-buildToolsVersion`|"34.0.0"|
|`GradleVersion`|"8.7"|
|`AndroidGradlePluginVersion`|"8.3.0"|
|`GradlePluginKotlinVersion`|"1.9.24"|
```

**Review by dpogue — APPROVED** [review]

**Comment by jcaron23** [comment]
@erisu The platform guide at https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html does not reflect the latest version.

I believe that should be patched in https://github.com/apache/cordova-docs/blob/master/www/docs/en/12.x/guide/platforms/android/index.md since the latest CLI version is still 12, right?

**Pull request merged**